lib/ReactViewModels/ViewState.ts

```diff
@@ -449,8 +447,6 @@ export default class ViewState {
       }
     );
 
-    this._disclaimerHandler = new DisclaimerHandler(terria, this);
```

Comment:
Is this not needed?

Reply:
This was used in v7 for showing catalog item initial message. In v8 this is handled here:

https://github.com/TerriaJS/terriajs/blob/d4ccee3f8476f7737f615ee06e8806f3d43ddd2b/lib/ModelMixins/MappableMixin.ts#L175-L179